libs/documentation/src/lib/components/popup/popup.module.ts

```diff
@@ -48,11 +57,12 @@ export const ROUTES = [
   imports: [
     CommonModule,
     DocumentationComponentsSharedModule,
-    PopupBasicModule
+    PopupBasicModule,
+    TooltipBasicModule
   ]
 })
 export class PopupModule {
   constructor(demoList: DocumentationDemoList) {
-    demoList.register('popup', DEMOS);
+    demoList.register('tooltip-popover', DEMOS);
```

Comment:
One of the folder structure rule that the stackblitz generation uses is that it looks at the name of the demo, in this case `tooltip-popover`, and expects it to be in a directory of the same name.  Since this was re-named, could you also rename the directory name from `popup` to `tooltip-popover`? You would need to also update the `path` values in this module file along with any imports - including in documentation.module